### ISSUE TITLE
ArgumentError string contains null byte

### ISSUE THREAD

**Issue by ajkiessl** [psu-libraries/researcher-metadata#389]
Someone submitted three ScholarSphere deposits, all of which were littered with these 'null bytes' (\u0000) in the abstract and subtitle fields.  It raises an error: 

https://app.bugsnag.com/west-arete/researcher-metadata-database/errors/61957d25aefe6600082bb7a2?filters[event.since]=30d&filters[error.status]=open&filters[app.release_stage]=production&event_id=61957d50008958008b0f0000

But the deposit was still successful and it updated all the data in the ScholarsphereWorkDeposits successfully:

https://metadata.libraries.psu.edu/admin/scholarsphere_work_deposit/473

Since this doesn't appear to be breaking anything, I'm considering it low-priority.  Although, it seems like it should be breaking something, but I can't figure out why it's not.

**Comment by awead** [comment]
ugh. This is probably a result of copy-pasta coming from the user.

**Comment by awead** [comment]
Note, this is not to reflect poorly on our users. They are innocent! It's the annoying UTF-8 encoding issues.

**Comment by EricDurante** [comment]
I don't think that this somehow worked in spite of the error. It looks to me like this user tried several times to deposit the same article over the course of several minutes. The first three attempts failed completely, the user would have seen a 500 error each time upon submitting the form, and nothing would have been saved to RMD or ScholarSphere. I think that they must have made a fourth attempt where they corrected the problematic input (props to them both for their persistence and for correctly guessing what was wrong with their submission), and then everything succeeded.

So things were breaking for sure, and then when they weren't I think it was because everything was actually ok.

So to avoid other occurrences of this, I guess we either need to rescue this kind of error and provide some feedback to the user, or we need to try to sanitize the input somehow before we save it.

**Comment by EricDurante** [comment]
@anaelizabethenriquez This would have been super frustrating, so it might be a good idea to reach out to this person if you haven't already heard from them just to assure them that we detected the problem that they were having and that we're going to work on fixing it.

**Comment by anaelizabethenriquez** [comment]
@EricDurante Thanks. We haven't heard from Nicholas Rowland, but I just sent him a note about this.
